ui/src/__tests__/MainText.test.tsx

```diff
@@ -0,0 +1,25 @@
+import * as Components from '@components';
+import { render, screen } from '@testing-library/react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+Enzyme.configure({ adapter: new Adapter() });
+const component = <Components.PublicationCreationMainText />;
```

Comment:
please move anything related to "MainText tests" in it's own test suite (in it's own "describe" method).